### ISSUE TITLE
[Bug] Can't build!

### ISSUE THREAD

**Issue by Emeraldnickel** [Goldorion/MCreatorFabricGenerator#113]
**Describe the bug**
I can't build my workspace, it just returns an error when I try.

**Log**
Executing 
    Gradle task: tasks

Build 
    info: MCreator 2021.1.18117, fabric-1.16.5, 64-bit, 8192 MB, Mac OS X, JVM 
    1.8.0_275, JAVA_HOME: 
    /Applications/MCreator.app/Contents/MacOS/jdk/Contents/Home/jre

> 
    Configure project :

Fabric 
    Loom: 0.6.58 Build(release #58)

:remapping 
    sources

FAILURE: 
    Build failed with an exception.

* 
    What went wrong:

A 
    problem occurred configuring root project 'ChromaticAbberations'.

> 
    Failed to notify project evaluation listener.

 
      > org/eclipse/core/runtime/IProgressMonitor has been compiled by a more 
    recent version of the Java Runtime (class file version 55.0), this version 
    of the Java Runtime only recognizes class file versions up to 52.0

* 
    Try:

Run 
    with --stacktrace option to get the stack trace. Run with --info or 
    --debug option to get more log output. Run with --scan to get full 
    insights.

* 
    Get more help at https://help.gradle.org

BUILD 
    FAILED in 2s

BUILD 
    FAILED

Task 
    completed in 2 seconds

**Affected version(s)** MCreator 2021.1

**To Reproduce**
I have no idea! It just... doesn't build correctly.

**Expected behavior**
What I expected was for it to actually build correctly...

**Screenshots**
No screenshots applicable

**Additional context**
No other context because I haven't touched anything with either MCreator or the plugin in a while

**Comment by Goldorion** [comment]
Can you give me your workspace, please? I will test when I will have some time.

**Comment by Emeraldnickel** [comment]
[ChromaticAbs.zip](https://github.com/Goldorion/Fabric-Generator-MCreator/files/6659945/ChromaticAbs.zip)

**Comment by ultimateimmersion** [comment]
I have the same error. Seeing this was 18 hour ago, it was working fine for me 10 hours ago but now that I got back to it it gives me the same Eclipse version error. Test playing works fine it's just when trying to export.

**Comment by Shroototem** [comment]
I have the same issue on building a workspace.

```
Executing 
    Gradle task: tasks

Build 
    info: MCreator 2021.1.18117, fabric-1.16.5, 64-bit, 16335 MB, Windows 10, 
    JVM 1.8.0_275, JAVA_HOME: C:\Program Files\Pylo\MCreator\jdk

> 
    Configure project :

Fabric 
    Loom: 0.6.58 Build(release #58)

:remapping 
    45 mods (TinyRemapper, intermediary -> named)

:remapping 
    sources

FAILURE: 
    Build failed with an exception.

* 
    What went wrong:

A 
    problem occurred configuring root project 'test'.

> 
    Failed to notify project evaluation listener.

 
      > org/eclipse/core/runtime/IAdaptable has been compiled by a more recent 
    version of the Java Runtime (class file version 55.0), this version of the 
    Java Runtime only recognizes class file versions up to 52.0

* 
    Try:

Run 
    with --stacktrace option to get the stack trace. Run with --info or 
    --debug option to get more log output. Run with --scan to get full 
    insights.

* 
    Get more help at https://help.gradle.org

BUILD 
    FAILED in 11s

BUILD 
    FAILED

Task 
    completed in 11 seconds
```

**Comment by eliteSchwein** [comment]
same here

**Comment by Goldorion** [comment]
Ok, thank you for all your reports. I will check why it’s happening.

**Comment by ultimateimmersion** [comment]
Is that something that can be fixed externally or will we have to wait for the next plugin update? Because it stopped working without plugin tampering/updating.

**Comment by Goldorion** [comment]
> 
> 
> Is that something that can be fixed externally or will we have to wait for the next plugin update? Because it stopped working without plugin tampering/updating.

I will have to do some tests because I don't know the exact cause of the problem. I have an idea, but I need to test it before giving news.

**Comment by ultimateimmersion** [comment]
I've heard it has something to do with JDK 11 and 8 mismatch where 11 is used by the upcoming MCR if I'm not mistaken.

**Comment by Goldorion** [comment]
> I've heard it has something to do with JDK 11 and 8 mismatch where 11 is used by the upcoming MCR if I'm not mistaken.

He’s using MCR 2021.1 with J8, so this is not the problem.

**Comment by Shroototem** [comment]
I had both JRE 8 and JDK 16 when I tried to build the workstation. I tested on the latest snapshot of MCreator and it seems to work.

**Comment by Goldorion** [comment]
> 
> 
> I had both JRE 8 and JDK 16 when I tried to build the workstation. I tested on the latest snapshot of MCreator and it seems to work.

Interesting thing, thank you for the info :)

**Comment by ultimateimmersion** [comment]
> I had both JRE 8 and JDK 16 when I tried to build the workstation. I tested on the latest snapshot of MCreator and it seems to work.

Interesting. I tried the snapshot as well but I got a ton of errors after building. I'll try again tho.

**Comment by Muugiwaara** [comment]
I found the right place, did you find a solution? I have the same problem

**Comment by Goldorion** [comment]
I've done some tests and I found the cause. I won't explain it in detail because almost nobody will understand, but to summarize the problem is caused by the JDK version of MCreator. In MCreator 2021.1, the JDK version used is J8u275. In this Java version, something is at the 52.0 version. However, in the Fabric loom (a Gradle plugin to make Fabric mods), something uses the 55.0 version. It's the reason why in MCreator 2021.2, there is no error. The JDK has been updated to J11, so we have the minimal required version. I can't fix the problem in the plugin as I don't have access to the JDK. The only solution for the moment is to use the 2021.2's snapshots. I'm sorry about this inconvenience. :/
